scripts/smeshr/tcdc_for_topo.sh

```diff
@@ -38,8 +38,11 @@ get_day() {
 
   TMPF=$DAY.grib2
 
+  echo "cat ${HRRR_PATTERN} | wgrib2 - -match "TCDC:entire atmosphere" -inv /dev/null -grib $TMPF"
```

Comment:
Do we still need those `echo` calls here and on line 44?

Reply:
Nope, we can remove these - I'll push that version in a bit

Reply:
Fixed in [3d36cac](https://github.com/UofU-Cryosphere/isnoda/pull/64/commits/3d36cacd308a6bab0fe010fb0b2ffe9d80994f1e)